test/verify/check-connection

```diff
@@ -1189,6 +1190,15 @@ UnixPath=/run/cockpit/session
         self.assertRegex(packages, r"(^|\n)base1\s+.*/usr/share/cockpit/base1")
         self.assertRegex(packages, r"(^|\n)system\s+.*/usr/share/cockpit/systemd")
 
+        bridges = m.execute("cockpit-bridge --bridges").strip()
+        bridges = json.loads(bridges)
+        self.assertTrue(len(bridges) > 0)
+        self.assertTrue(any(b['name'] in ['sudo', 'pkexec'] for b in bridges))
```

Comment:
That'd be assertIn(), see CodeQL complaint

Reply:
Ah, double filtering, but you can just assert sudo -- all our images should have that. They shuold also all have pkexec, but that's not the point of this test, so sudo is enough IMHO.

Reply:
I wasn't sure of we would have `pkexec` or `sudo` by default.. But uh how would `assertIn` work for pcp? I'd have to look up the full path which is kinda janky

Reply:
Again: I have *no idea* what kind of information this command is supposed to show -- pcp was just a thought. If it shows priv bridges, then testing for sudo is just fine IMHO.

Reply:
It shows all birdges from the manifests, so checking if pcp is in there seems reasonable, as it's optional.